public/components/query_compare/search_result/search_components/__tests__/__snapshots__/search_configs.test.tsx.snap

```diff
@@ -372,7 +362,7 @@ exports[`Flyout component Renders flyout component 1`] = `
                           }
                           showPrintMargin={false}
                           tabSize={2}
-                          theme="sql_console"
+                          theme="textmate"
```

Comment:
Change to `textmate` should've been merged in, may need a rebase? Or did this come up only from this PR?